server/server.js

```diff
@@ -150,68 +150,85 @@ app.route('/login')
-        res.redirect('/login')
+        res.redirect("/login")
         return
     }
-    else if(typeof req.params.installationId !== "string") {
+      
+    try {
+        let allInstallations = await db.getInstallations()
+            
+        let viewParams = {
+            installations: allInstallations
+                .filter((installation) => installation.isActive)
+                .map((installation) => {
+                    return { name: installation.name, id: installation.id }
+                }),
+        };
+        viewParams.viewMessage = allInstallations.length >= 1 ? "Please select an installation" : "No installations to display"
+      
+        res.send(Mustache.render(chatbotDashboardTemplate, viewParams))
+    } catch (err) {
+        helpers.log(err)
+        res.status(500).send()
+    }
+})
+      
+app.get("/dashboard/:installationId?", async (req, res) => {
+    if (!req.session.user || !req.cookies.user_sid) {
+        res.redirect("/login")
+        return
+    } else if (typeof req.params.installationId !== "string") {
         let installations = await db.getInstallations()
-        res.redirect('/dashboard/' + installations[0].id)
+        res.redirect("/dashboard/" + installations[0].id)
         return
     }
-
+      
     try {
-        let recentSessions = await db.getRecentSessionsWithInstallationId(req.params.installationId)
-        let currentInstallation = await db.getInstallationWithInstallationId(req.params.installationId)
+        let recentSessions = await db.getRecentSessionsWithInstallationId(
+            req.params.installationId
+        )
+        let currentInstallation = await db.getInstallationWithInstallationId(
+            req.params.installationId
+        )
         let allInstallations = await db.getInstallations()
-        
+      
         let viewParams = {
             recentSessions: [],
             currentInstallationName: currentInstallation.name,
             installations: allInstallations
-                .filter(installation => installation.isActive)
-                .map(installation => { 
+                .filter((installation) => installation.isActive)
+                .map((installation) => {
                     return { name: installation.name, id: installation.id }
-                })
-        }
-
-        for(const recentSession of recentSessions) {
+                }),
+        };
+      
+        for (const recentSession of recentSessions) {
             let createdAt = moment(recentSession.createdAt, moment.ISO_8601)
             let updatedAt = moment(recentSession.updatedAt, moment.ISO_8601)
             viewParams.recentSessions.push({
                 unit: recentSession.unit,
-                createdAt: createdAt.tz('America/Vancouver').format('DD MMM Y, hh:mm:ss A'),
-                updatedAt: updatedAt.tz('America/Vancouver').format('DD MMM Y, hh:mm:ss A'),
+                createdAt: createdAt
+                    .tz("America/Vancouver")
```

Comment:
Hmmm, looks like you might have accidentally run an auto-formatter on this file. Is that what changed a bunch of these quote marks and new lines?

Reply:
Yeah I think I formatted it, and then only undid the indentation - will fix!